lib/__tests__/dom.nested.test.js

```diff
@@ -51,7 +51,9 @@ describe('DocumentMeta - DOM nested', () => {
       <div>
         <DocumentMeta {...DOC_META} />
         <div>
-          <DocumentMeta {...DOC_META_NESTED} extend />
+          <DocumentMeta {...DOC_META_NESTED} extend>
+            <div>Hello world!</div>
```

Comment:
> DocumentMeta is NOT supposed to have nested DOM elements.

Oh, I thought it was supposed to based on [this](https://github.com/kodyl/react-document-meta/blob/master/lib/index.js#L201-L203). Either way, It’s more convenient/clear to nest your DOM inside `<DocumentMeta>` and have it as the root.
```js
  render: function render () {
    return this.props.children ? React.Children.only(this.props.children) : null;
  }
```

> DocumentMeta does not render anything to your DOM

Not true. When used as sibling, `<DocumentMeta>` returns a `<noscript>` element. A React component must always return [one single child](https://facebook.github.io/react/docs/top-level-api.html#react.createclass).